CRM/Core/BAO/CustomValueTable.php

```diff
@@ -91,16 +91,6 @@ public static function create($customParams, $parentOperation = NULL) {
                 $value = 'null';
                 break;
               }
-
-              // need to add/update civicrm_entity_file
-              $entityFileDAO = new CRM_Core_DAO_EntityFile();
-              $entityFileDAO->file_id = $field['file_id'];
-              $entityFileDAO->find(TRUE);
-
-              $entityFileDAO->entity_table = $field['table_name'];
-              $entityFileDAO->entity_id = $field['entity_id'];
-              $entityFileDAO->file_id = $field['file_id'];
-              $entityFileDAO->save();
```

Comment:
There are still places that create links with `eid=` in them, and probably also in universe, and for any file created after this that link seems to now give a 404, because the `eid=` in the url is blank.

Reply:
@demeritcowboy this PR doesn't affect the links, that was done in https://github.com/civicrm/civicrm-core/pull/32290

Can you give steps to reproduce that problem so I can look into it?

Reply:
1. Create a custom field of type file for cases.
2. Upload a file to that field. It's maybe slightly easier to see the problem if it's an image file.
3. After saving, note the file link gives 404, and that eid= is blank in the url.

This is working in master without the PR.

Reply:
@demeritcowboy good catch. That's more related to the other PR so I made a fix for 6.1: https://github.com/civicrm/civicrm-core/pull/32326